metapool/sample_sheet.py

```diff
@@ -1820,21 +1836,24 @@ def make_sample_sheet(metadata, table, sequencer, lanes, strict=None):
         If a value is omitted from this dictionary the values in square
         brackets are used as defaults.
 
-        - Bioinformatics: List of dictionaries describing each project's
-          attributes: Sample_Project, QiitaID, BarcodesAreRC, ForwardAdapter,
-          ReverseAdapter, HumanFiltering, library_construction_protocol,
-          experiment_design_description
+        - Bioinformatics: List of dictionaries, one per project, describing
```

Comment:
Cleaning the docstring a little so it has more info; note that these changes are based on my limited experience using it.